Cargo.toml

```diff
@@ -1,13 +1,10 @@
 [package]
-authors = [
-    "Carl Lerche <me@carllerche.com>",
-    "Andrew Cann <shum@canndrew.org>",
-]
+authors = ["MaidSafe Developers <dev@maidsafe.net>"]
 description = "tokio-based uTP implementation"
 documentation = "https://docs.rs/tokio-utp"
-license = "MIT/Apache-2.0"
-name = "tokio-utp"
-repository = "https://github.com/canndrew/tokio-utp"
+license = "MIT/BSD"
```

Comment:
This line should be
```
license = "MIT OR BSD-3-Clause"
```

Reply:
so basically pls use whatever is there in `safe_app` @StephenCoyleAtMaidSafe , to be consistent

Reply:
updated 👍